### PR TITLE
User checking aa

### PR DESCRIPTION
**Pull request by ArtemAnoshin**

### CODE REVIEW

File: js/ct-checkusers.js [hunk 1]
```diff
@@ -48,9 +48,18 @@ window.apbct = window.apbct || {};
     usersChecker.setListeners = () => {
         $('#check_spam_users').click(() => {
             usersChecker.improvedCheck = $("#ct_impspamcheck_checkbox").is(":checked");
-            usersChecker.clearUserCheckerResults().then(() => {
+
+            usersChecker.clearUserCheckerResults()
+                .then((response) => {
+                    // Clear Frontend Results
+                    const responseObject = JSON.parse(response);
```
Comment (Glomberg) [inline js/ct-checkusers.js]: `JSON.parse()` has to be wrapped by `try...catch`

File: js/ct-checkusers.js [hunk 2]
```diff
@@ -135,6 +144,7 @@ window.apbct = window.apbct || {};
     };
 
     usersChecker.runUserCheckerSuccess = (response) => {
+        usersChecker.reactUserCheckingFrontendData(response?.checkingCount, response?.foundedSpam);
```
Comment (Glomberg) [inline js/ct-checkusers.js]: Is there a way not using the `optional chaining '?.'` functioning ?

**Review by Glomberg — CHANGES_REQUESTED** [review]

File: js/ct-checkusers.js (hunk 1 above)
Comment (ArtemAnoshin) [reply js/ct-checkusers.js]: DONE

File: js/ct-checkusers.js (hunk 2 above)
Comment (ArtemAnoshin) [reply js/ct-checkusers.js]: DONE

**Review by Glomberg — APPROVED** [review]

**Review by svfcode — APPROVED** [review]

**Pull request merged**